### ISSUE TITLE
Crash in multiplayer [Bug]

### ISSUE THREAD

**Issue by Taknax** [TeamLapen/Vampirism#812]
Server thread/WARN [vampirism]: [EntityEventHandler]Could not replace zombie target task

**Comment by maxanier** [comment]
A) pretty sure this is not a crash, it is probably logged because another mod messes with zombie AI. the game should work fine, however, zombies might attack vampires.
B) please provide a crash log, if it actually is one
C) Don't delete the form when creating an issue but fill it in

**Comment by Taknax** [comment]
Ty for reply and this beautiful mod. Im gonna check and try to fix.

Is it possible the issue is come fron champions mod? It tieres the mobs.

**Comment by Taknax** [comment]
Now i have crash report after fighting a wizard from the Wizardry mod. I didnt understand how a wizardry mod can influence vampirism?
I figured out that IPixelies Gender Mod is causing the most trouble. After deleting the issue got better, but not fully gone.

[https://pastebin.com/cT92TAQX](https://pastebin.com/cT92TAQX)

Edit: I tried without Vampirism and the issue was gone.

One log more: [https://pastebin.com/VYXDsTaf](url)
